### DISCUSSION TITLE
Using a point cloud

### DISCUSSION THREAD

**Discussion by slanteri** [lululxvi/deepxde#1171] [Q&A]
Dear Lu Lu,

I am trying to solve a 2d time-dependent PDE with DeepXDE. 

For the sampling of spatial points in a 2d domain, I am using a point cloud with:
    **spatial_domain =  dde.geometry.PointCloud(points=xy_dom, boundary_points=xy_brd**)

Then for the definition of the spatio-temporal geometry, I simply set:
    **time_domain  = dde.geometry.TimeDomain(t_lower, t_upper)**
    **geomtime  = dde.geometry.GeometryXTime(spatial_domain, time_domain)**

My questions are the following:
- What will be the sampling of the time dimension? Is there a way to impose this sampling?
- When  defining the PDE problem and data set with **data = dde.data.TimePDE(geomtime, pde, ...)**  
  what value should I use for **num_domain** (number of points in the domain) ? 
  Currently I am using the number of values (x,y coordinates) in **xy_dom** but I guess that I am missing the 
  number of points in the time dimension ?

Thank you in advance for your help.

**Comment by lululxvi** [comment]
- See https://deepxde.readthedocs.io/en/latest/modules/deepxde.data.html#deepxde.data.pde.PDE . By default, train_distribution='Hammersley'
- You can use any value you like for `num_domain`
